index.html

```diff
@@ -5,9 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <title>mobileFirst</title>
 </head>
 <body>
+    <nav>
+        <header>My Logo</header>
+        <img src="Images/menu-button.png" alt="">  
+    </nav>
+    <main>
+            <h1>I'm Joseph Glad to meet you!</h1>  
+        <p>
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+            <p class="contain">LET’S CONNECT</p>
+        
+        <div class="social"> 
+            <img src="Images/twitter.svg" alt="">
+            <img src="Images/Linkedin icon.svg" alt="">
+            <img src="Images/m vector.svg" alt="">
+            <img src="Images/github.svg" alt="">
+            <img src="Images/Vector.svg" alt="">
```

Comment:
- [ ] Please as the last reviewer said these icons represent an unordered list of elements they should all be placed inside a `ul`  tag and each of them inside a `li` and then inside an `a` tag since they represent a link to a website.

Reply:
These have been worked on as requested.

Reply:
hi @Ddiiro maybe you forget to push the changes for the list elements 😊 I see that you implement them now good job